src/parser/mod.rs

```diff
@@ -16510,6 +16511,27 @@ impl<'a> Parser<'a> {
         }
     }
 
+    fn parse_export(&mut self) -> Result<Statement, ParserError> {
```

Comment:
```suggestion
    fn parse_export_date(&mut self) -> Result<Statement, ParserError> {
```
Could we also add a short description mentioning that this parses a `Statement::ExportData` statement? [For example](https://github.com/apache/datafusion-sqlparser-rs/blob/1a7e8d969ae509d99f5f9aa1136092da002a77c6/src/parser/mod.rs#L659-L661)